src/components/FilterPopup/FilterPopup.tsx

```diff
@@ -0,0 +1,114 @@
+import { useRef } from 'react';
+import styled from 'styled-components';
+import { useClickOutside, useKeyboard, KeyCode, Popup, Button, Tabs, TabContent, TabsMenu } from '@taskany/bricks';
+import { gray10, gapS, gapXs, textColor, gapM, radiusM } from '@taskany/colors';
+import { IconBulbOnOutline } from '@taskany/icons';
+
+import { Tip } from '../Tip';
+
+import { tr } from './FilterPopup.i18n';
+
+interface FilterPopupProps {
+    visible: boolean;
+    switchVisible: (val: boolean) => void;
+    filterRef: React.RefObject<HTMLSpanElement>;
+    onApplyClick: () => void;
+    activeTab?: string;
+}
+
+const StyledPopupWrapper = styled.div`
+    padding: ${gapXs};
+`;
+
+const StyledTabs = styled(Tabs)`
+    width: 500px;
+    height: 200px;
+
+    gap: ${gapM};
+
+    ${TabsMenu} {
+        width: 200px;
+        max-width: 100%;
+        flex-basis: auto;
+    }
+
+    ${TabContent} {
+        display: flex;
+        flex-direction: column;
+        overflow: auto;
+        box-sizing: border-box;
+        flex: 1;
+    }
+`;
+
+const StyledPopup = styled(Popup)`
+    border-radius: ${radiusM};
```

Comment:
Why not defaults? Popup already has border radius

Reply:
In figma screen the popup have same a corner radius